src/web-worker/amp-worker.js

```diff
@@ -72,12 +72,40 @@ class AmpWorker {
     // Use RTV to make sure we fetch prod/canary/experiment correctly.
     const useLocal = getMode().localDev || getMode().test;
     const useRtvVersion = !useLocal;
-    const url = calculateEntryPointScriptUrl(
-      loc,
-      'ww',
-      useLocal,
-      useRtvVersion
-    );
+
+    let url = '';
+
+    const policy = {
+      createScriptURL: function (url) {
+        // Only allow the correct webworker url to pass through
+        const regexURL =
+          /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org(\/.*)?$/;
+        const testRegexURL = /^([a-zA-Z0-9_-]+\.)?localhost$/;
```

Comment:
getting a `testRegexURL` is unused error

Reply:
Right, I had removed it from the conditional to see if the checks are failing due to the limitations on the test url. I found that the tests are using urls other than localhost including: "http://0.0.0.0:8000, http://127.0.0.1:8000 and http://192.168.192.3:8000" I will likely update testRegexURL to whitelist those additional URLs. I'm not sure why the module tests are failing though.